### ISSUE TITLE
Payments cannot connect to React Dev Tool

### ISSUE THREAD

**Issue by LZoog** [mozilla/fxa#3208]
We should be able to use React Dev Tools in the payments server for easier development and debugging - it shows a view of the component tree, state, component props, and highlights rerenders. I see this message when clicking on the extension:

<img width="428" alt="image" src="https://user-images.githubusercontent.com/13018240/67957863-3dc51a00-fbc4-11e9-8145-b12c749222d9.png">

Looks like we may need to add it as a project dependency to work in our case (perhaps because we work with an iFrame), see "Usage with React DOM": https://github.com/facebook/react/tree/master/packages/react-devtools

Let's avoid ejecting to customize Webpack (or for anything else) if at all possible.

**Comment by lmorchard** [comment]
Starting to dig into this a bit, and I think it's our CSP restrictions that are blocking [the `window.__REACT_DEVTOOLS_GLOBAL_HOOK__` property](https://github.com/facebook/react/blob/master/packages/react-devtools-extensions/src/injectGlobalHook.js) that the React Dev Tools are looking for:
![image](https://user-images.githubusercontent.com/21687/68159293-5c9a1800-ff06-11e9-8bf5-1f44b4cd55cc.png)

**Comment by lmorchard** [comment]
Yeah, when [I disable CSP altogether on my local server](https://github.com/mozilla/fxa/blob/master/packages/fxa-payments-server/server/config/index.js#L36), I get React Dev Tools working:

![image](https://user-images.githubusercontent.com/21687/68159760-3fb21480-ff07-11e9-8d44-63e32aa97e33.png)

Looks like [this is probably a Firefox-specific bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1267027#c48) where our CSP blocks the script injected by an extension.
